sass/helpers/spacing/margin-padding.sass

```diff
@@ -26,4 +26,11 @@ $current: $range-start
     margin-#{$side}: 0 !important
   
   .is-#{$side}-paddingless
-    padding-#{$side}: 0 !important
\ No newline at end of file
+    padding-#{$side}: 0 !important
+
+@each $side in (left, right)
+  has-margin-#{$side}-auto
```

Comment:
Duplicated code with lines 35-36. Plz just remove lines 32-33, because they aren't definition of a new class anyway (because of lack of dot before class name). Thx! :slightly_smiling_face:

Reply:
hi, jmaczan, thank for the review. 😄